### ISSUE TITLE
Apply softmax to logits

### ISSUE THREAD

**Issue by figuernd** [WHOIGit/ros-triton-classifier#5]
The response from Triton includes negative values and so it appears Triton does not apply any kind of activation function to the output. Unfortunately we only get back the number of classes requested so we don't have the entire set of raw logits to get a proper softmax.
